pkgs/applications/display-managers/ly/default.nix

```diff
@@ -1,19 +1,20 @@
-{ stdenv, lib, fetchFromGitHub, linux-pam }:
+{ stdenv, lib, fetchFromGitHub, git, linux-pam, libxcb }:
 
 stdenv.mkDerivation rec {
   pname = "ly";
-  version = "0.2.1";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
-    owner = "cylgom";
+    owner = "fairyglade";
     repo = "ly";
-    rev = version;
-    sha256 = "16gjcrd4a6i4x8q8iwlgdildm7cpdsja8z22pf2izdm6rwfki97d";
+    rev = "v${version}";
+    sha256 = "1vpcdswdr50qnpphvfywck6ksxb7qmljbml4hjy0hsdx6blc7igg";
     fetchSubmodules = true;
   };
 
-  buildInputs = [ linux-pam ];
-  makeFlags = [ "FLAGS=-Wno-error" ];
+  hardeningDisable = [ "all" ];
+  nativeBuildInputs = [ git ];
+  buildInputs = [ libxcb linux-pam ];
```

Comment:
Shouldn't `libxcb` be in `nativeBuildInputs` ?

Reply:
As far as I understand it, `native*` contain packages from which binaries need to be run during the respective phases. That's not the case with `libxcb`, it's just a library that `ly` compiles against.